packages-next/fields/src/types/relationship/views/index.tsx

```diff
@@ -43,18 +49,15 @@ function LinkToRelatedItems({
           .map(({ id }) => id)
           .join(',')}"`}
       >
-        View List of Related Items
+        View related items
       </Button>
     );
   }
+
+  // QUESTION: why is the target `_blank` here?
```

Comment:
I believe it's because that's how it is in the current Admin UI - I don't have opinions on whether it should be target _blank or not

The reason it's not using the router Link is because it's target _blank so it wouldn't do anything